bench/axis_real_world.go

```
@@ -152,6 +152,61 @@ func afterEach(hooks ...func(*Sandbox, Observation) error) func(*Sandbox, Observ
+		}
+		var collection statusEnvelope
+		if err := decodeWaveObservation(collected, &collection, "intended-untracked collection STATUS"); err != nil {
+			return err
+		}
+		if collection.Authority.LineageID != "" || collection.NextTransition.Kind != "collect" || len(collection.NextTransition.Collect.Inputs) != 1 || collection.NextTransition.Collect.Inputs[0].Name != "intended_untracked_selection" {
+			return fmt.Errorf("incomplete START left authority or no selection boundary: %+v", collection.NextTransition)
+		}
+		var inventory []string
+		digest := collection.argument("expected_untracked_inventory")
+		if err := json.Unmarshal([]byte(collection.argument("eligible_paths_json")), &inventory); err != nil || len(inventory) != wantInventory || digest == "" {
+			return fmt.Errorf("eligible inventory = %d paths, digest %q, parse error %v; want %d", len(inventory), digest, err, wantInventory)
```

golangci-lint (2.12.2)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)